pyfr/solvers/navstokes/kernels/avis.mako

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+<%inherit file='base'/>
+<%namespace module='pyfr.backends.base.makoutil' name='pyfr'/>
+
+<%
+    import math
```

Comment:
I think you can use mako to do the import; `<%namespace module='math' name='math'/>`